### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by dieum512**
In this branch I:

- Used the Webpack I created in the exercise,
- Created the HTML for my project,
- Styled my project with CSS, 
- Check and Corrected some linter errors and
- Updated the README file.

Thanks for your time to review this work

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @dieum512, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the table of contents or any headers in it's README file
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="to-do-body">
+          <form class="first-form" action="">
+            <input
+              type="text"
+              class="add-task"
+              placeholder="Add to your list..."
+            />
+            <button class="add-btn">Add</button>
+          </form>
+          <ul class="to-do-list">
```
Comment (Olamarx) [inline src/index.html]: - [ ] Kindly note that when a project is required to be displayed dynamically, it means it should be rendered from the js file. Therefore, I will advise you to remove all the added html tags and create the data in an array like 

```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

Then you can iterate over it with a loop by doing something like 

```
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

### Note, it does not have to be like that, I just used that as an example of dynamic display or dynamic rendering.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @dieum512, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file ✔️ 
- Webpack config ✔️ 
- Styling ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ ✔️ 

Hi @dieum512, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**